Twittimation/RandomLikeAutomaton.cs

```diff
@@ -0,0 +1,104 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Twittimation.Commands;
+using Twittimation.Http;
+using Twittimation.IO;
+using Twittimation.Twitter;
+
+namespace Twittimation
+{
+    public class RandomLikeAutomaton : IAutomaton
+    {
+        private readonly IStored<RandomLikeAutomatonData> _data;
+        private readonly IStored<Tasks> _tasks;
+        private readonly Like _like;
+        private readonly IStored<Credentials> _credentials;
+        private int _updatesToSkip = 0;
+        private const int SecondsPerHour = 60 * 60;
+
+        public RandomLikeAutomaton(IStored<RandomLikeAutomatonData> data, IStored<Tasks> tasks, Like like, IStored<Credentials> credentials)
+        {
+            _data = data;
+            _tasks = tasks;
+            _like = like;
+            _credentials = credentials;
+        }
+
+        public void Update()
```

Comment:
This method is extremely long. Should be refactored such that no sub-method is longer than 10 lines.